src/js/actions/search/menucommands.js

```diff
@@ -53,7 +53,13 @@ define(function (require, exports) {
             }
 
             if (labels[part].$MENU) {
-                path += labels[part].$MENU + ">";
+                // Don't show "Application" options on Windows
```

Comment:
I don't think any of this is necessary? There is no "application" menu on Windows...